core/src/main/java/org/testcontainers/images/RemoteDockerImage.java

```diff
@@ -38,10 +38,16 @@
     @ToString.Exclude
     private DockerClient dockerClient = DockerClientFactory.lazyClient();
 
+    public RemoteDockerImage(DockerImageName dockerImageName) {
+        this.imageNameFuture = CompletableFuture.completedFuture(dockerImageName);
+    }
+
+    @Deprecated
```

Comment:
Since the contract was always that `RemoteDockerImage` accepts a full image name, perhaps we can keep this and `(String,String)` constructors?

Otherwise, it becomes a bit too long:
```java
new GenericContainer(new RemoveDockerImage(new DockerImageName("missed-me:1"))
```

Reply:
Will defer as per https://github.com/testcontainers/testcontainers-java/pull/2839#discussion_r449549759